src/kOS.Safe/Execution/CPU.cs

```diff
@@ -15,15 +15,18 @@ namespace kOS.Safe.Execution
 {
     public class CPU : ICpu
     {
-        private enum Status
+        private enum Section
         {
-            Running = 1,
-            Waiting = 2
+            Main = 1,
+            Trigger = 2
         }
```

Comment:
Yeah that one is tricky because the interpreter ALSO has a separate main and trigger.  (you can type a "when" command into the interpreter and it *will* trigger later on provided you're in the interpreter and not in the middle of a program.)

In fact that's why I didn't call it "context".  We already have a thing called "context" and it is that - the difference between being in the interpreter or in a program is the ProgramContext.